packages/visual-editor/src/utils/applyHeaderScript.ts

```diff
@@ -0,0 +1,13 @@
+export const applyHeaderScript = (document: Record<string, any>) => {
+  if (!document?.__?.visualEditorConfig) {
+    return;
+  }
+
+  const headerScript: string = JSON.parse(
+    document.__.visualEditorConfig
+  )?.headerScript;
+
+  if (headerScript) {
+    return `<script>${headerScript}</script>`;
```

Comment:
Don't include `<script>`. The user needs to do this in Storm. Just return `headerScript` directly.